### PR TITLE
Add `npm install` to instructions

### PR DESCRIPTION
**Pull request by jordancoblin**
Confusing error message if you run `npm start` before running `npm install`.

### CODE REVIEW

**Comment by jordancoblin** [comment]
@justin0022 😬

**Comment by justin0022** [comment]
The setup instructions in the main readme already has the installation instructions - but it may be good to reiterate. Could you modify to something along the lines of if you see a weird error message, try npm install first?

File: 1-Canvas-API-Token.md [hunk 1]
```diff
@@ -28,7 +28,8 @@
 > Never share your token with anyone. If you think your token may have been exposed (for example, by accidentally posting it to GitHub), delete your token from Canvas right away. Instructions for creating and deleting access tokens as a student are [available on the Canvas Guides](https://community.canvaslms.com/docs/DOC-16005-42121018197).
 
 ## Start the server
-1. Type `npm start` into terminal in `backend` directory. If your server is already running, kill it first: `control + c`, then start the server.
+1. Navigate to the `backend` directory and run `npm install` to install the backend dependencies.
```
Comment (justin0022) [inline 1-Canvas-API-Token.md]: This is where the installation instructions are in the main readme: https://github.com/UBC-LA-Hackathon/student-dashboard#%EF%B8%8F-getting-started

But ya I think having it here again is a good plan!

Comment (jordancoblin) [reply 1-Canvas-API-Token.md]: Updated!

**Review by justin0022 — APPROVED** [review]
lgtm!

**Pull request merged**